chore: Add a pre-release issue template (#775)
stackabletech/docker-images: .github/ISSUE_TEMPLATE/pre-release.md

---

> name: Pre-Release Product Updates

Reviewer: suggestion:

```suggestion
name: Pre-Release Container Image Updates
```

> about: This template can be used to track the product leading up to the next Stackable release

Reviewer: note: Seems like there is something missing in that sentence.

```suggestion
about: This template can be used to track the container image updates leading up to the next Stackable release
```

> title: Release XX.(X)X

Reviewer: suggestion:

```suggestion
title: "chore: Update Container Images for Stackable Release XX.(X)X"
```

Author: Oh I like this!

labels: ['epic']
assignees: ''
---

<!--
    DO NOT REMOVE THIS COMMENT. It is intended for people who might copy/paste from the previous release issue.
    This was created by an issue template: https://github.com/stackabletech/issues/issues/new/choose.
-->

> ## Product Updates for Stackable Release XX.(X)X

Reviewer: suggestion:

```suggestion
## Container Image Updates for Stackable Release XX.(X)X
```
